src/screens/SSIContactsOverviewScreen/index.tsx

```diff
@@ -74,7 +75,7 @@ class SSIContactsOverviewScreen extends PureComponent<IProps, IState> {
     };
     const style = {
       ...backgroundStyle,
-      ...(itemInfo.index === contacts.length - 1 && itemInfo.index % 2 !== 0 && {borderBottomWidth: 1, borderBottomColor: borderColors.dark}),
+      // ...(itemInfo.index === contacts.length - 1 && itemInfo.index % 2 !== 0 && {borderBottomWidth: 1, borderBottomColor: borderColors.dark}),
```

Comment:
why has this been disabled?

Reply:
i am unsure whether the new contact designs require a bottom border on the last item of the list. but i have added this bottom border back in now. i dont see that it would be an issue to leave it in.